src/Controller/UserProfilController.php

```diff
@@ -30,4 +30,30 @@ public function showProfil(): Response
             'coachInfo' => $coachInfo,
         ]);
     }
+
+    /**
+     * @Route("/{id}/edit", name="profil_edit", methods={"GET","POST"})
```

Comment:
c'est le user qui edit son propre profile, c'est comme pour la vue /prolie, là ta vue /profile/edit ne doit pas avoir d'id, car car tu peux récup direct $this->getUser dans le controller pour avoir le user actuellement connecté

Reply:
Je ne savais pas comment récupérer les données...mais oui le $this->getUser ça fonctionne bien...merci :) c'est modifié